### DISCUSSION TITLE
Call by 'first name' or by 'last name' in translations?

### DISCUSSION THREAD

**Discussion by Altonss** [CatimaLoyalty/Android#658] [General]
For some languages like french or german for example, the translation of you can be done in too ways. Either a more familiar approach or more polite one (eg. more first name or last name). What way would you prefer?

**Comment by TheLastProject** [comment]
I don't really have any preference, I think it depends on the culture itself. In fact, Dutch in the Netherlands is generally more informal but Dutch in Belgium is more polite.

I do think it is good to make a decision and use it as a translator note though.

**Reply by Altonss** [reply]
Yes things are often cultural. And therefor I am a little bit lost for the moment :) But I will try to use the same style as in the already translated strings and it should be fine for now :)

**Reply by TheLastProject** [reply]
That sounds like the best way to go about it yeah :)
